test/runnable/test42.d

```diff
@@ -52,7 +50,7 @@ void test2()
 void test3()
 {
     auto i = mixin("__LINE__");
-    writefln("%d", i);
+    printf("%d\n", i);
     assert(i == 54);
```

Comment:
This has changed

Reply:
Please merge once this is fixed.

Reply:
??

Reply:
I don't get it. What's wrong?

Reply:
```
auto i = mixin("__LINE__"); // line 52 therefore i == 52
...
assert(i == 54);
```

Reply:
Ah, silly me. Thanks!

Reply:
Im sure you would have found it eventually.